tests/test_sourmash.py

```diff
@@ -348,8 +351,8 @@ def test_do_compare_output_multiple_k(c):
 def test_do_compare_output_multiple_moltype(c):
     testdata1 = utils.get_test_data('short.fa')
     testdata2 = utils.get_test_data('short2.fa')
-    c.run_sourmash('compute', '-k', '21', '--dna', testdata1)
-    c.run_sourmash('compute', '-k', '63', '--no-dna', '--protein', testdata2)
```

Comment:
Why did the `63` has changed to `21`? Was it wrong in the first place? thanks.

Reply:
I'm not sure why I changed `63` to `21` but it was probably because the test wasn't passing. I can recheck and change it though. :)

Reply:
I'm not sure if it was wrong in the first place.

Reply:
great question! this changed in 4.0.

per [release notes](https://github.com/dib-lab/sourmash/releases/tag/v4.0.0), changed behavior:

* protein ksizes in MinHash are now divided by 3, except in sourmash compute (#1277)

Reply:
That makes sense! Thanks @ctb